### PR TITLE
Course edits file #1

### PR DESCRIPTION
**Pull request by StavC**
* Reviewed the installation process in a clean state Win11, everything works just fine.
* Edited some steps in the Readme.md file for better clarity.

### CODE REVIEW

**Review by cnoam — CHANGES_REQUESTED** [review]
I think you forgot to add the assets dir. 
When adding, make sure the link names are relative (and not "stav/assets" )

**Comment by StavC** [comment]
> I think you forgot to add the assets dir. When adding, make sure the link names are relative (and not "stav/assets" )

should be fine now

**Comment by StavC** [comment]
Just added two mini exercises to the end of notebook 04 Spark SQL.

File: readme.md [hunk 1]
```diff
@@ -163,6 +164,12 @@ If the above doesn't work:
  1. close all browser tab of the jupyter notebooks
  1. run `docker compose restart`
 
+## the 'run' command returns "docker: command not found:
```
Comment (cnoam) [inline readme.md:167]: the --> The

**Comment by cnoam** [comment]
great work!

**Pull request merged**